### ISSUE TITLE
Add demographic variables in embedding LSTMs

### ISSUE THREAD

**Issue by tschimbr** [eonum/medcodelearn#20]

**Comment by tschimbr** [comment]
First convert model to Graph

**Comment by tschimbr** [comment]
Either use demographic variables as tokens in a sequential model or add them directly in a graph model.

**Comment by tschimbr** [comment]
Demographic variables as tokens (See #25 )
Test on PDX and DRG: + 1.5%

**Comment by tschimbr** [comment]
Wait until Keras 1.0 is out with the new Graph API.

**Comment by tschimbr** [comment]
See https://github.com/eonum/medcodelearn/tree/demovars
Acc before with demographic tokens: 48.07%
Acc before without demographic tokens: 47.27%
Acc after with demographic tokens: 48.47%
Acc after without demographic tokens: 47.64%

The addition of an additional dense layer before the final softmax output layer did not result in any improvements.
